src/pages/PlanPage/subRoutes/UpgradePlanPage/UpgradeForm/UpgradeForm.jsx

```diff
@@ -78,6 +79,7 @@ function UpgradeForm({ selectedPlan, setSelectedPlan }) {
       </div>
       <PlanTypeOptions
         setFormValue={setFormValue}
+        getFormValues={getFormValues}
```

Comment:
Might not be needed after this https://github.com/codecov/gazebo/pull/2533/files#r1468071697